Build error on Debian: make[1]: *** [debian/rules:42: override_dh_auto_build] Error 1

See https://github.com/jellyfin/jellyfin/discussions/5215 (just in case someone commented there, the issue text is identical)

While the `dotnet ...` build process seems to finish without problems the package is not built, the process fails in `override_dh_auto_build`:

```
...
dh binary
   dh_update_autotools_config
   dh_autoreconf
   debian/rules override_dh_auto_build
make[1]: Entering directory '/home/frank/Projects/Jellyfin/jellyfin-server-10.8.0'
dotnet publish --configuration Release --output='/home/frank/Projects/Jellyfin/jellyfin-server-10.8.0/usr/lib/jellyfin/bin' --self-contained --runtime debian-x64 \
	"-p:DebugSymbols=false;DebugType=none" Jellyfin.Server
Microsoft (R) Build Engine version 16.8.3+39993bd9d for .NET
Copyright (C) Microsoft Corporation. All rights reserved.

  Determining projects to restore...
  All projects are up-to-date for restore.
make[1]: *** [debian/rules:42: override_dh_auto_build] Error 1
make[1]: Leaving directory '/home/frank/Projects/Jellyfin/jellyfin-server-10.8.0'
make: *** [debian/rules:33: binary] Error 2
dpkg-buildpackage: error: fakeroot debian/rules binary subprocess returned exit status 2
```

Manually running `dotnet` seems to work, the process finishes with status 0:

```
frank@build:~/Projects/Jellyfin/jellyfin-server-10.8.0$ dotnet publish --configuration Release --output='/home/frank/Projects/Jellyfin/jellyfin-server-10.8.0/usr/lib/jellyfin/bin' --self-contained --runtime debian-x64 \
        "-p:DebugSymbols=false;DebugType=none" Jellyfin.Server
Microsoft (R) Build Engine version 16.8.3+39993bd9d for .NET
Copyright (C) Microsoft Corporation. All rights reserved.

  Determining projects to restore...
  All projects are up-to-date for restore.
... (the thing builds...)
frank@build:~/Projects/Jellyfin/jellyfin-server-10.8.0$ echo $?
0
```

Still `override_dh_auto_build` fails. What gives? Am I missing something or is the build broken at this moment?

Building on *Debian bullseye/sid* using `dotnet-sdk-5.0 5.0.103-1` on x86_64

--- a/debian/rules
+++ b/debian/rules
@@ -39,7 +39,7 @@ override_dh_auto_test:
 override_dh_clistrip:
 
 override_dh_auto_build:
-	dotnet publish --configuration $(CONFIG) --output='$(CURDIR)/usr/lib/jellyfin/bin' --self-contained --runtime $(DOTNETRUNTIME) \
+	dotnet publish -maxcpucount:1 --configuration $(CONFIG) --output='$(CURDIR)/usr/lib/jellyfin/bin' --self-contained --runtime $(DOTNETRUNTIME) \
 		"-p:DebugSymbols=false;DebugType=none" Jellyfin.Server
 
 override_dh_auto_clean:

